src/plugins/gui/scenes/SepaFormScene.tsx

```diff
@@ -57,21 +57,31 @@ export const SepaFormScene = React.memo((props: Props) => {
   }, [])
 
   return (
-    <SceneWrapper background="theme">
-      <KeyboardAwareScrollView keyboardShouldPersistTaps="handled" extraScrollHeight={theme.rem(2.75)} enableAutomaticScroll enableOnAndroid>
-        <SceneHeader title={headerTitle} underline withTopMargin />
-        <EdgeText style={styles.formSectionTitle}>{lstrings.bank_info_title}</EdgeText>
-        <GuiFormField fieldType="name" value={name} label={lstrings.form_field_title_account_owner} onChangeText={handleNameInput} autofocus />
-        <GuiFormField fieldType="iban" value={iban} label={lstrings.form_field_title_iban} onChangeText={handleIbanInput} />
-        <GuiFormField fieldType="swift" value={swift} returnKeyType="done" label={lstrings.form_field_title_swift_bic} onChangeText={handleSwiftInput} />
-        <MainButton
-          label={lstrings.string_next_capitalized}
-          marginRem={[1, 0.5, 1, 0.5]}
-          type="secondary"
-          disabled={!name.trim() || !iban.trim() || !swift.trim()}
-          onPress={handleSubmit}
-        />
-      </KeyboardAwareScrollView>
+    <SceneWrapper background="theme" hasNotifications hasOverscroll={false}>
+      {gap => (
+        <>
+          <KeyboardAwareScrollView
+            keyboardShouldPersistTaps="handled"
+            extraScrollHeight={theme.rem(2.75)}
+            enableAutomaticScroll
+            enableOnAndroid
+            contentContainerStyle={{ paddingBottom: gap.bottom }}
```

Comment:
When we add `contentContainerStyle={{ paddingBottom: gap.bottom }}`, we should also do `style={ marginBottom: -gap.bottom }`, so the scroll view extends underneath the notification area.

If that looks wrong, then we need to fix the `SceneWrapper` to use the correct values.

Reply:
Made the correct updates across the board. It was a bit confusing, there is a
distinction between style padding/marginBottom vs contentContainerStyle
paddingBottom (overscroll)